app/views/profiles/show.erb

```diff
@@ -0,0 +1,207 @@
+<% @fields = current_organization.partner_form_fields %>
```

Comment:
Not sure if this got lost but @seanmarcia what are your thoughts on making this into a partial and putting it in the `partners#show` page?

Reply:
Sorry I missed this comment. You mean put all the profile info on the show page? I thought about it but it feels like a pretty full page already especially for the banks that have a lot of notes and prior distributions on a partner. Maybe when Steph has time to go through everything she can think of a good way to display all the relevant stuff or have a better flow?

Reply:
@seanmarcia yep, I suggested putting that information there since I think we have a problem with discoverability. In my mind, I'd rather have them say they find it annoying and cluttered than asking us where the information is (this happens a lot). 

What do you say about adding it there and waiting to hear about an issue about it then we can use Steph's expertise to add it in there?